election/election_test.go

```diff
@@ -409,7 +409,7 @@ func TestResign(t *testing.T) {
 
 	// Calling resign on a follower should have no effect
 	err := c1.Nodes["n1"].Node.Resign(context.Background())
-	require.NoError(t, err)
+	assert.ErrorContains(t, err, "not the leader")
```

Comment:
Is it a fix for a broken test?

Reply:
The red line was added in previous PR, then I realized it broke the test.  The error was expected.